src/app/api/services/workflow/workflow.service.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@angular/core';
 import { CoreWebService } from 'dotcms-js/dotcms-js';
 import { Observable } from 'rxjs/Observable';
 import { RequestMethod } from '@angular/http';
-import { Workflow } from '../../../shared/models/workflow/workflow.model';
+import { DotWorkflowActions } from '../../../shared/models/dot-workflow-actions/dot-workflow-actions.model';
```

Comment:
I should be ```DotWorkflowAction``` because the interface represent one, is many when is an array of ```DotWorkflowAction``` same with the file name.